src/en/news/blog/2022/centralized_logging/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Centralized Logging on Ceph Dashboard"
+date: "2022-11-23"
+author: "Nizamudeen A"
+image: "images/banner.jpg"
+tags:
+  - "ceph"
+  - "dashboard"
+  - "logs"
+  - "monitoring"
+  - "loki"
+  - "promtail"
+  - "grafana"
+  - "centralized logging"
+---
+
+_Disclaimer: This article proclaims Loki as a hero._
+
+## What are we trying to solve?
+
+Ceph, as a distributed system, usually generates a lot of logs. And these logs can be from mons, mgr, osds or any of the clients of ceph.
+
+Naturally, we look at the logs to get some detailed information regarding the events that happened throughout the cluster lifecycle.
+Logs are what we immediately look for if we find something unusual going on in the cluster. Or if you like reading, its a fun place to be 😜.
+In anycase, we'll want to check through logs from different clients and they are not available from a single source. Its all distributed as it is intended.
+
+## Enter Loki.
+
+Thankfully, the Loki we are dealing with is not an Asgardian God(imagine the chaos). Our Loki is a log aggregation system which is designed to store and
+query the logs. Infact it is not just Loki, it is [Grafana Loki](https://grafana.com/oss/loki/).
+
+We have a Promtail too.
+
+![tony_stark_promtail](images/we_have_promtail.png)
+
+[Promtail](https://grafana.com/docs/loki/latest/clients/promtail/) is an agent which ships the contents of local logs to a private Grafana Loki instance.
+
+Let's make it simple enough. We introduced a feature called Centralized Logging, where we can go to a particular section in the Ceph Dashboard and see the
+logs from all the clients in just one place. Cool, isn't it?
+
+## How to set it up?
+
+First, we have to make sure the grafana is configured on our cluster.
+
+Then, go to the **Cluster > Services** page, and **Create** the two services - Loki and Promtail.
+
+![services_list](images/loki_promtail.png)
+
+If successful, we can see all the daemons are **running**.
+
+##### Status of Loki Daemon
+
+![loki_daemon](images/loki.png)
+
+##### Status of Promtail Daemons
+
+![promtail_daemon](images/promtail.png)
+
+Now we need to enable logging to the files. For that, we can go to **Cluster > Configurations**,
+search for the **log_to_file** and **mon_cluster_log_to_file**, in the search area
+
+![log_to_file](images/log_to_file.png)
+
+Just select and **Edit** the option. And select the value of **global** to true
+
+![config_form](images/form_config.png)
+
+We have to do the same for **mon_cluster_log_to_file** as well.
+
+![mon_cluster_log_to_file](images/mon_log_to_file.png)
+
+If you can't find the item you searched for, then you can just **clear the filter** and it will bring you the configuration option that you searched.
```

Comment:
```suggestion
If you can't find the item you searched for, then you can just **clear the filter** and it will bring you the configuration option that you searched for.
```